jekyll/install_unix.md

```diff
@@ -7,32 +7,126 @@ current: Install
 
 <h1 class="text-centered page-title main-heading">Install Nim on Unix</h1>
 
+There are several ways to install Nim on Unix-like systems, depending on
+your needs:
+
+* If you just want to use the stable version of Nim, you may find it simplest
+  to use your system's package manager.
+
+* If you want to easily switch between versions of Nim,
+  including switching between the stable version and the development
+  nightlies, you may prefer ``choosenim``.
```

Comment:
I'd like to see every bullet point having a link to the section about it.

E.g.
```
... [system's package manager](#Installation-using-package-managers)
... [``choosenim``](#Installation-using-choosenim)
...  [manual installation from source](#Manual-installation-from-source)
```

EDIT:
These links seem to be working on my markdown preview, but not on the site.... :/

Reply:
Done.